src/components/MapView/AlertForm/index.tsx

```diff
@@ -0,0 +1,330 @@
+import React, { useMemo, useState } from 'react';
+import {
+  Box,
+  Button,
+  createStyles,
+  FormControl,
+  ListSubheader,
+  MenuItem,
+  Select,
+  TextField,
+  Theme,
+  Typography,
+  withStyles,
+  WithStyles,
+} from '@material-ui/core';
+import { faBell, faCaretDown } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { useSelector } from 'react-redux';
+import { getBoundaryLayerSingleton } from '../../../config/utils';
+import { BoundaryLayerProps, LayerKey } from '../../../config/types';
+import { LayerData } from '../../../context/layers/layer-data';
+import { layerDataSelector } from '../../../context/mapStateSlice/selectors';
+import LayerDropdown from '../Layers/LayerDropdown';
+
+// Not fully RFC-compliant, but should filter out obviously-invalid emails.
+const EMAIL_REGEX: RegExp = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+
+// This should probably be determined on a case-by-case basis,
+// depending on if the downstream API has the capability.
+// For now it can be permanently enabled.
+const ALERT_FORM_ENABLED = true;
+
+enum Statistic {
```

Comment:
Is it possible to use this: https://github.com/WFP-VAM/prism-frontend/blob/6b51b51ba9532a25614fd35395f55c02fb1455e6/src/config/types.ts#L219
And add `max` and `min` to it?

If the analysis tool doesn't support max and min, you could split those into a new enum and make:
```ts
enum MaxMin{
Max='max',Min='min'
}
type Statistic= MaxMin | AggregationOption;
```